packages/workflow/src/Interfaces.ts

```diff
@@ -985,8 +983,7 @@ export type IExecuteFunctions = ExecuteFunctions.GetNodeParameterFn &
 			BinaryHelperFunctions &
 			DeduplicationHelperFunctions &
 			FileSystemHelperFunctions &
-			SSHTunnelFunctions &
-			JsonHelperFunctions & {
+			SSHTunnelFunctions & {
```

Comment:
Are we sure we are not breaking anything (e.g. custom nodes) by removing `JsonHelperFunctions`?

Reply:
we did not remove it, we merged it in `BaseHelperFunctions`. This is all backward compatible.